docs/user/firststeps.rst

```diff
@@ -228,9 +235,78 @@ For example, if you want to keep the new notebook(s), run the following.
     git commit -m "Added some notebooks"
     git push
 
-Record you work and make it repeatable
+
+.. _interactive_exploration:
+
+Interactively explore the bicycle data
 """"""""""""""""""""""""""""""""""""""
 
+To start working with the bicycle data we have already created a sample
+notebook that does some data cleaning and visualization. We will first
+download the notebook so you can interactively explore the dataset, much like
+you would in a real project. Feel free to execute the cells. When we are ready
+to generate results, we will refactor the code from the notebook into a python
+module and run it with ``renku`` to create a repeatable analysis workflow.
+
+Use the commands below to add the notebook to your project.
+
+.. code-block:: console
+
+    mkdir -p notebooks
+    wget -O "notebooks/zhbikes-notebook.ipynb" https://raw.githubusercontent.com/SwissDataScienceCenter/renku/master/docs/_static/zhbikes/ZHBikes.ipynb
+    git add notebooks
+    git commit -m"Added zuerich bike notebook"
+    git push
+
+
+Refactor the notebook
+"""""""""""""""""""""
+
+To make our work here more reusable and easier to maintain we will refactor
+the code we have written in the notebook into runnable python scripts. We will
+make two scripts: one that does some initial preprocessing of the data and
+saves the result, and another that will create the figures.
+
+Here, we have already done the refactoring work for you - to get the scripts,
+run:
+
+.. code-block:: console
+
+    mkdir -p src
+    wget -O "src/clean_data.py" https://raw.githubusercontent.com/SwissDataScienceCenter/renku/master/docs/_static/zhbikes/clean_data.py
+    wget -O "src/plot_data.py" https://raw.githubusercontent.com/SwissDataScienceCenter/renku/master/docs/_static/zhbikes/plot_data.py
+
```

Comment:
![screenshot_cleandata](https://user-images.githubusercontent.com/17312721/44984425-f53cad00-af7c-11e8-8623-1fe72d9ed3c4.png)
Computer says no. The file is not properly downloaded and there is no explanation. The other file downloaded fine. No error messages..I tried again a few times with the same result. Ideas?

Reply:
I guess: replace `master` with `revise-firststeps`

Reply:
Please note that I have done that already.

Reply:
I don't understand, is the link incorrect?

Reply:
I copied the link by hand from the repo so I don't think so. I don't know what is going on, but what happened is that the file starts to download and then stops. Result: the file downloaded partially. I am not sure what the problem was here. I tried several times yesterday and kept getting a partially broken file. This morning I tried and the file was downloaded properly. Maybe something in the environment? 
For now: moving on.

Reply:
Ok thanks, I'll try as well

Reply:
`wget https://raw.githubusercontent.com/SwissDataScienceCenter/renku/revise-firststeps/docs/_static/zhbikes/clean_data.py` worked for me

Reply:
I think it was a network/cdn fluke yesterday